store/tikv/mock-tikv/mvcc.go

```diff
@@ -95,9 +95,9 @@ func (e *mvccEntry) lockErr() error {
 	}
 }
 
-func (e *mvccEntry) Get(ts uint64) ([]byte, error) {
-	if e.lock != nil {
-		if e.lock.startTS <= ts {
+func (e *mvccEntry) Get(ts uint64, isoLevel kvrpcpb.IsolationLevel) ([]byte, error) {
+	if isoLevel == kvrpcpb.IsolationLevel_SI {
+		if e.lock != nil && e.lock.startTS <= ts {
 			return nil, e.lockErr()
```

Comment:
Why not `if isoLevel == kvrpcpb.IsolationLevel_SI && if e.lock != nil && e.lock.startTS <= ts`?